nucypher/config/constants.py

```diff
@@ -62,8 +62,9 @@
 # Web
 CLI_ROOT = NUCYPHER_PACKAGE / 'network' / 'templates'
 TEMPLATES_DIR = CLI_ROOT / 'templates'
+STATICS_DIR = os.getenv('NUCYPHER_STATICS_DIR') or os.path.join(
```

Comment:
You can try your hand at `pathlib.Path` like the other constants above 😄 - https://docs.python.org/3/library/pathlib.html. Example,
```python
pathlib.Path(__file__) / 'statics'
```